unblob/handlers/compression/bzip2.py

```diff
@@ -81,7 +81,7 @@ def bzip2_recover(self, file: io.BufferedIOBase, start_offset: int) -> int:
                 # This can be negative, but we don't care, because we also count found blocks
                 # and in the case we didn't found both of them, we don't use this value
                 # When there are two blocks found, this will reflect the correct end
-                current_block_end = bits_read - (COMPRESSED_MAGIC_LENGTH + 1)
+                current_block_end = bits_read - COMPRESSED_MAGIC_LENGTH
 
                 curr_block += 1
```

Comment:
This `curr_block` value is never used.

Reply:
Just saw that myself while I am rewriting the code further :)